jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-security-pull.yaml

```diff
@@ -55,8 +55,7 @@
         git clone https://github.com/kubernetes/test-infra -b master
         './test-infra/jenkins/bootstrap.py' \
             --job='{job-name}' \
-            --pull="${{PULL_REFS}}" \
-            --repo='{repo-name}' \
+            "--repo=k8s.io/{repo-name}=${{PULL_REFS}}" \
```

Comment:
(I feel like the security repo is not under k8s.io, and I also feel like the presubmit for security repo is broken (which might be good?) )

Reply:
whoops! fixed. probably pasted from the kubernetes pull config :roll_eyes: